src/lib/fetcher/fetcher.ts

```diff
@@ -76,12 +76,17 @@ export default class Fetcher<T> {
       .map((_, i) => i)
       .filter(i => results[i] instanceof Error);
 
-    results.map(result => {
-      this.logger.info(
-        'Results Data:',
-        JSON.stringify((result as any).data, null, 4).substring(0, 500),
-      );
-    });
+    results
+      .filter(result => !(result instanceof Error))
+      .map(result => {
+        debugger;
+        this.logger.info(
+          'Results Data:',
+          JSON.stringify((result as any).data)
+            .replace(/(?:\r\n|\r|\n)/g, ' ')
+            .substring(0, 1000),
```

Comment:
The json string will not break and become unparseable in CloudWatch because it replaces line feeds and reduces the length to 1000 characters. It is just a single log recorded by CloudWatch.